tests/partitions.t

```diff
@@ -0,0 +1,33 @@
+Partitions command success:
+
+  $  ringpop-admin partitions 127.0.0.1:3000
+   Checksum     # Nodes   Sample Host    
+   *   5         127.0.0.1:3000  (glob)
+
+
+With bootstrap file:
+
+  $  ringpop-admin partitions $TESTDIR/hosts.json
+   Checksum     # Nodes   Sample Host    
+   *   5         127.0.0.1:3000  (glob)
+
+
+Unable to connect to host:
+
+  $  ringpop-admin partitions 127.0.0.1:2999
```

Comment:
Yes. We could glob all the lines I guess. Or improve the error output to be more human-readable. Also, note that it's very easy to update the test if the output changes (just do `./run-tests --update` and you can accept any changes you know are valid output).